components/assetlist/stories/assetlist.stories.js

```diff
@@ -63,7 +72,6 @@ Default.args = {
 			label: "Downloads",
 			isSelectable: true,
 			isBranch: true,
-			isSelected: false,
```

Comment:
removing this now renders that `Set boolean` button in the control panel. Did we want to keep it for now?

<img width="535" height="72" alt="Screenshot 2025-10-22 at 1 39 49 PM" src="https://github.com/user-attachments/assets/b2ab7773-2ba5-485d-9196-92a571e52035" />